server/index.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const cors = require('cors');
+const proxy = require('http-proxy-middleware');
+const ejs = require('ejs');
+
+const port = process.env.PORT || 4000;
+
+const app = express();
+
+app.use(cors());
+app.use(express.static('public'));
+
+app.engine('html', ejs.renderFile);
+app.get('/:id', (req, res) => {
+  res.render('../public/index.html');
+});
+app.set('view engine', 'html');
+
+
+app.use(proxy('/api/experiences', {target: 'http://localhost:3001/' }));
```

Comment:
And seems that we did not use these 'proxy' lib also... But this is not to say that there is a problem, just another solution.

Reply:
For some reason, my proxy server did not work without the proxy library so I kept it in. Thanks.